Rudder-Appsflyer/Classes/RudderAppsflyerIntegration.m

```diff
@@ -156,37 +209,37 @@ - (void) addCheckoutProperties: (NSDictionary *) properties params: (NSMutableDi
         [params setValue:productCategories forKey:AFEventParamContentType];
         [params setValue:productQuants forKey:AFEventParamQuantity];
         
-        NSString *currency = properties[@"currency"];
+        NSString *currency = properties[KeyCurrency];
         if (currency != nil) {
             [params setValue:currency forKey:AFEventParamCurrency];
         }
         
-        NSString *orderId = properties[@"order_id"];
+        NSString *orderId = properties[KeyOrderId];
         if (orderId != nil) {
             [params setValue:orderId forKey:AFEventParamReceiptId];
```

Comment:
Shouldn't `AFEventParamReceiptId` be `transaction_id`?

Reply:
No, AppsFlyer standard params say that this should be linked to af_order_id and af_receipt_id, please refer [here](https://support.appsflyer.com/hc/en-us/articles/360019347957#purchase-af_purchase) for more details about the same

Reply:
Cool.